Simplenote/TagListViewController.swift

```diff
@@ -860,7 +860,7 @@ private struct Localization {
     static let done = NSLocalizedString("Done", comment: "Done editing tags")
 
     static let allNotes = NSLocalizedString("All Notes", comment: "All Notes filter button")
-    static let trash = NSLocalizedString("Trash-noun", comment: "Trash filter button")
+    static let trash = NSLocalizedString("Trash", comment: "Trash filter button")
```

Comment:
```suggestion
    static let trash = NSLocalizedString("Trash", comment: "Trash filter button (noun)")
```

There might be an issue with this one, as it seems like `NSLocalizedString("Trash", "Trash filter button")` is also used elsewhere in the code in a different context, at least based on the fact that `genstrings` [aggregates not only the comment for this one, but for another as well ("Title: Trash Tag is selected") in the generated `.strings` file](https://github.com/Automattic/simplenote-ios/pull/1443/files#diff-b008231bd522825760f4af589e37ded40359d336760494c92cfe707ef0417429R822-R823).

So we're using the same English term for a noun here and a title elsewhere, while in different locales the term for those two contexts might be different.

I don't have a solution for this though 🤷‍♂️  (that's a long-known issue of our process and way we handle localization in iOS with English copy as the key… which is something I wish we could change, especially to avoid this kind of issues, but it would be quite a big change and it's probably too late at this point anyway). The only hack I can think of to distinguish them is to add a dummy special character (or even just a space, assuming they're not trimmed by GlotPress?) in the key/text, and remove it at runtime… but this would feel quite dirty…

Reply:
EDIT: btw for now I think it's ok to go with this (even if this means 2 different contexts for the same string) for now.
I'm hoping to address that kind of edge case with some refactoring of our tooling when I get to tackle our Localization Project (ref: paaHJt-sd-p2) soon.

_(TL;DR the likely idea being to separate the source copies in `Base.lproj` and have `en.lproj` be an actual locale in GlotPress, so that we can have a key (in `NSLocalizedString` code and in `Base.lproj` extracted by `genstrings`) that is slightly different from the English copy when it is really needed. At least that's the idea that I'm likely to propose and write a RFC about when I get to that)_